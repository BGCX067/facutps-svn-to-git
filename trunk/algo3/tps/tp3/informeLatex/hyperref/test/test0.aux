\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\@writefile{toc}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {section}{\numberline {}{ABSTRACT}}{E1}{section*.2}}
\@writefile{toc}{\contentsline {section}{\numberline {1}A first, simple, section heading}{E1}{section.1}}
\@writefile{toc}{\contentsline {section}{\numberline {2}A funny \textsf  {ANSYS}\xspace  -section with a $\qopname  \relax o{log}$}{E1}{section.2}}
\newlabel{page2}{{2}{E2}{A funny \ANS -section with a \texorpdfstring {$\log $}{log}\relax }{section.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Yet another ?`\c C\"asar M\"uller n\~ao!`}{E3}{section.3}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Testing Int\'ernal J\oe mps}{E3}{section.4}}
\newlabel{ss:intjmps}{{4}{E3}{Testing Int\'ernal J\oe mps\relax }{section.4}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}A \ss ection with $\leq $}{E4}{section.5}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {5.0.1}A subsubsection $a+b$}{E4}{subsubsection.5.0.1}}
\@writefile{toc}{\contentsline {paragraph}{\numberline {5.0.1.1}A paragraph}{E4}{paragraph.5.0.1.1}}
\@writefile{toc}{\contentsline {subparagraph}{A subparagraph}{E4}{section*.3}}
\@writefile{toc}{\contentsline {section}{\numberline {6}Testing External Jumps}{E5}{section.6}}
\newlabel{ss:extjmps}{{6}{E5}{Testing External Jumps\relax }{section.6}{}}
\newlabel{LastPage}{{}{E5}{}{page.E5}{}}
