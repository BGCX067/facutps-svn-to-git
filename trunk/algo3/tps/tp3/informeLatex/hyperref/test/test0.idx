\indexentry{an item on page 3|hyperpage}{E3}
