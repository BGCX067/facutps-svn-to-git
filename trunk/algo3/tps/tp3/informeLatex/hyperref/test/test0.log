This is pdfTeXk, Version 3.141592-1.40.3 (Web2C 7.5.6) (format=pdflatex 2009.5.6)  4 JUN 2009 00:38
entering extended mode
 Source specials enabled.
 file:line:error style messages enabled.
 %&-line parsing enabled.
**test0
(./test0.tex
LaTeX2e <2005/12/01>
Babel <v3.8h> and hyphenation patterns for english, usenglishmax, dumylang, noh
yphenation, loaded.
(/usr/share/texmf-texlive/tex/latex/base/article.cls
Document Class: article 2005/09/16 v1.4f Standard LaTeX document class
(/usr/share/texmf-texlive/tex/latex/base/size10.clo
File: size10.clo 2005/09/16 v1.4f Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
)
(/usr/share/texmf-texlive/tex/latex/graphics/color.sty
Package: color 2005/11/14 v1.0j Standard LaTeX Color (DPC)

(/etc/texmf/tex/latex/config/color.cfg
File: color.cfg 2007/01/18 v1.5 color configuration of teTeX/TeXLive
)
Package color Info: Driver file: pdftex.def on input line 130.

(/usr/share/texmf-texlive/tex/latex/pdftex-def/pdftex.def
File: pdftex.def 2007/01/08 v0.04d Graphics/color for pdfTeX
\Gread@gobject=\count87
))
(/usr/share/texmf-texlive/tex/latex/graphics/graphicx.sty
Package: graphicx 1999/02/16 v1.0f Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf-texlive/tex/latex/graphics/keyval.sty
Package: keyval 1999/03/16 v1.13 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/share/texmf-texlive/tex/latex/graphics/graphics.sty
Package: graphics 2006/02/20 v1.0o Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texmf-texlive/tex/latex/graphics/trig.sty
Package: trig 1999/03/16 v1.09 sin cos tan (DPC)
)
(/etc/texmf/tex/latex/config/graphics.cfg
File: graphics.cfg 2007/01/18 v1.5 graphics configuration of teTeX/TeXLive
)
Package graphics Info: Driver file: pdftex.def on input line 90.
)
\Gin@req@height=\dimen103
\Gin@req@width=\dimen104
)
(/usr/share/texmf-texlive/tex/latex/lastpage/lastpage.sty
Package: lastpage 1994/06/25 v0.1b LaTeX2e package for refs to last page number
 (JPG)
)
(/usr/share/texmf-texlive/tex/latex/tools/xspace.sty
Package: xspace 2006/05/08 v1.12 Space after command names (DPC,MH)
)
(/usr/share/texmf-texlive/tex/latex/base/makeidx.sty
Package: makeidx 2000/03/29 v1.0m Standard LaTeX package
)
(/usr/share/texmf-texlive/tex/latex/amsmath/amsmath.sty
Package: amsmath 2000/07/18 v2.13 AMS math features
\@mathmargin=\skip43

For additional information on amsmath, use the `?' option.
(/usr/share/texmf-texlive/tex/latex/amsmath/amstext.sty
Package: amstext 2000/06/29 v2.01

(/usr/share/texmf-texlive/tex/latex/amsmath/amsgen.sty
File: amsgen.sty 1999/11/30 v2.0
\@emptytoks=\toks15
\ex@=\dimen105
))
(/usr/share/texmf-texlive/tex/latex/amsmath/amsbsy.sty
Package: amsbsy 1999/11/29 v1.2d
\pmbraise@=\dimen106
)
(/usr/share/texmf-texlive/tex/latex/amsmath/amsopn.sty
Package: amsopn 1999/12/14 v2.01 operator names
)
\inf@bad=\count88
LaTeX Info: Redefining \frac on input line 211.
\uproot@=\count89
\leftroot@=\count90
LaTeX Info: Redefining \overline on input line 307.
\classnum@=\count91
\DOTSCASE@=\count92
LaTeX Info: Redefining \ldots on input line 379.
LaTeX Info: Redefining \dots on input line 382.
LaTeX Info: Redefining \cdots on input line 467.
\Mathstrutbox@=\box26
\strutbox@=\box27
\big@size=\dimen107
LaTeX Font Info:    Redeclaring font encoding OML on input line 567.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 568.
\macc@depth=\count93
\c@MaxMatrixCols=\count94
\dotsspace@=\muskip10
\c@parentequation=\count95
\dspbrk@lvl=\count96
\tag@help=\toks16
\row@=\count97
\column@=\count98
\maxfields@=\count99
\andhelp@=\toks17
\eqnshift@=\dimen108
\alignsep@=\dimen109
\tagshift@=\dimen110
\tagwidth@=\dimen111
\totwidth@=\dimen112
\lineht@=\dimen113
\@envbody=\toks18
\multlinegap=\skip44
\multlinetaggap=\skip45
\mathdisplay@stack=\toks19
LaTeX Info: Redefining \[ on input line 2666.
LaTeX Info: Redefining \] on input line 2667.
)
(/usr/share/texmf-texlive/tex/latex/hyperref/hyperref.sty
Package: hyperref 2007/02/07 v6.75r Hypertext links for LaTeX
\@linkdim=\dimen114
\Hy@linkcounter=\count100
\Hy@pagecounter=\count101

(/usr/share/texmf-texlive/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2007/02/07 v6.75r Hyperref: PDFDocEncoding definition (HO)
)
(/etc/texmf/tex/latex/config/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
(/usr/share/texmf-texlive/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2006/08/22 v2.4 Connects package keyval with LaTeX options (
HO)
)

Package hyperref Warning: Values of option `pdfpagemode':
(hyperref)                * `UseNone'
(hyperref)                * `UseOutlines'
(hyperref)                * `UseThumbs'
(hyperref)                * `FullScreen'
(hyperref)                * `UseOC' (PDF 1.5)
(hyperref)                * `UseAttachments' (PDF 1.6)
(hyperref)                * An empty value disables the option.
(hyperref)                Unknown value `None' on input line 2238.

Package hyperref Info: Option `plainpages' set `false' on input line 2238.
Package hyperref Info: Hyper figures OFF on input line 2288.
Package hyperref Info: Link nesting OFF on input line 2293.
Package hyperref Info: Hyper index ON on input line 2296.
Package hyperref Info: Plain pages OFF on input line 2303.
Package hyperref Info: Backreferencing OFF on input line 2308.
Implicit mode ON; LaTeX internals redefined
Package hyperref Info: Bookmarks ON on input line 2444.
(/usr/share/texmf-texlive/tex/latex/ltxmisc/url.sty
\Urlmuskip=\muskip11
Package: url 2005/06/27  ver 3.2  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 2599.
\Fld@menulength=\count102
\Field@Width=\dimen115
\Fld@charsize=\dimen116
\Choice@toks=\toks20
\Field@toks=\toks21
Package hyperref Info: Hyper figures OFF on input line 3102.
Package hyperref Info: Link nesting OFF on input line 3107.
Package hyperref Info: Hyper index ON on input line 3110.
Package hyperref Info: backreferencing OFF on input line 3117.
Package hyperref Info: Link coloring OFF on input line 3122.
\Hy@abspage=\count103
\c@Item=\count104
\c@Hfootnote=\count105
)
*hyperref using default driver hpdftex*
(/usr/share/texmf-texlive/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2007/02/07 v6.75r Hyperref driver for pdfTeX
\Fld@listcount=\count106
)
\@indexfile=\write3
\openout3 = `test0.idx'.


Writing index file test0.idx
No file test0.aux.
\openout1 = `test0.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 16.
LaTeX Font Info:    ... okay on input line 16.
Package hyperref Info: Link coloring OFF on input line 16.
(/usr/share/texmf-texlive/tex/latex/hyperref/nameref.sty
Package: nameref 2006/12/27 v2.28 Cross-referencing by name of section

(/usr/share/texmf-texlive/tex/latex/oberdiek/refcount.sty
Package: refcount 2006/02/20 v3.0 Data extraction from references (HO)
)
\c@section@level=\count107
)
LaTeX Info: Redefining \ref on input line 16.
LaTeX Info: Redefining \pageref on input line 16.
\@outlinefile=\write4
\openout4 = `test0.out'.

\tf@toc=\write5
\openout5 = `test0.toc'.



LaTeX Warning: Reference `page2' on page E1 undefined on input line 24.

[1

{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}] [2] [3]

LaTeX Warning: Reference `ss:intjmps' on page E4 undefined on input line 49.


Package hyperref Warning: Token not allowed in a PDFDocEncoded string,
(hyperref)                removing `math shift' on input line 51.


Package hyperref Warning: Token not allowed in a PDFDocEncoded string,
(hyperref)                removing `\leq' on input line 51.


Package hyperref Warning: Token not allowed in a PDFDocEncoded string,
(hyperref)                removing `math shift' on input line 51.


Package hyperref Warning: Difference (2) between bookmark levels is greater 
(hyperref)                than one, level fixed on input line 54.


Package hyperref Warning: Difference (2) between bookmark levels is greater 
(hyperref)                than one, level fixed on input line 57.

<hog.pdf, id=39, 251.94125pt x 213.79875pt>
File: hog.pdf Graphic file (type pdf)
 <use hog.pdf>
File: hog.pdf Graphic file (type pdf)
 <use hog.pdf>
File: hog.pdf Graphic file (type pdf)

<use hog.pdf> [4 <./hog.pdf>]
File: hog.pdf Graphic file (type pdf)
 <use hog.pdf>
File: hog.pdf Graphic file (type pdf)
 <use hog.pdf>
No file test0.ind.
[5] AED: lastpage setting LastPage (./test0.aux)

LaTeX Warning: There were undefined references.


LaTeX Warning: Label(s) may have changed. Rerun to get cross-references right.

 ) 
Here is how much of TeX's memory you used:
 3145 strings out of 95086
 42458 string characters out of 1183256
 98606 words of memory out of 1500000
 6293 multiletter control sequences out of 10000+50000
 6140 words of font info for 23 fonts, out of 1200000 for 2000
 28 hyphenation exceptions out of 8191
 27i,7n,39p,299b,337s stack positions out of 5000i,500n,6000p,200000b,5000s
</usr/share/texmf-texlive/fonts/type1/bluesky/cm/cmbx10.pfb></usr/share/texmf
-texlive/fonts/type1/bluesky/cm/cmbx12.pfb></usr/share/texmf-texlive/fonts/type
1/bluesky/cm/cmex10.pfb></usr/share/texmf-texlive/fonts/type1/bluesky/cm/cmmi10
.pfb></usr/share/texmf-texlive/fonts/type1/bluesky/cm/cmr10.pfb></usr/share/tex
mf-texlive/fonts/type1/bluesky/cm/cmr12.pfb></usr/share/texmf-texlive/fonts/typ
e1/bluesky/cm/cmssbx10.pfb></usr/share/texmf-texlive/fonts/type1/bluesky/cm/cms
y10.pfb>
Output written on test0.pdf (5 pages, 55718 bytes).
PDF statistics:
 99 PDF objects out of 1000 (max. 8388607)
 20 named destinations out of 1000 (max. 131072)
 22 words of extra memory for PDF output out of 10000 (max. 10000000)

