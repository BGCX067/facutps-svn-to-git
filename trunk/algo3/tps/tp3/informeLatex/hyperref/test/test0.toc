\addvspace {10pt}
\contentsline {section}{\numberline {}{ABSTRACT}}{E1}{section*.2}
\contentsline {section}{\numberline {1}A first, simple, section heading}{E1}{section.1}
\contentsline {section}{\numberline {2}A funny \textsf {ANSYS}\xspace -section with a $\qopname \relax o{log}$}{E1}{section.2}
\contentsline {section}{\numberline {3}Yet another ?`\c C\"asar M\"uller n\~ao!`}{E3}{section.3}
\contentsline {section}{\numberline {4}Testing Int\'ernal J\oe mps}{E3}{section.4}
\contentsline {section}{\numberline {5}A \ss ection with $\leq $}{E4}{section.5}
\contentsline {subsubsection}{\numberline {5.0.1}A subsubsection $a+b$}{E4}{subsubsection.5.0.1}
\contentsline {paragraph}{\numberline {5.0.1.1}A paragraph}{E4}{paragraph.5.0.1.1}
\contentsline {subparagraph}{A subparagraph}{E4}{section*.3}
\contentsline {section}{\numberline {6}Testing External Jumps}{E5}{section.6}
