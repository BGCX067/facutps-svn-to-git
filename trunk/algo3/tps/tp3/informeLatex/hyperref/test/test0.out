\BOOKMARK [1][-]{section*.2}{ABSTRACT}{}
\BOOKMARK [1][-]{section.1}{A first, simple, section heading}{}
\BOOKMARK [1][-]{section.2}{A funny ANSYS-section with a log}{}
\BOOKMARK [1][-]{section.3}{Yet another \277\307\344sar M\374ller n\343o\241}{}
\BOOKMARK [1][-]{section.4}{Testing Int\351rnal J\234mps}{}
\BOOKMARK [1][-]{section.5}{A \337ection with }{}
\BOOKMARK [2][-]{subsubsection.5.0.1}{A subsubsection a+b}{section.5}
\BOOKMARK [3][-]{paragraph.5.0.1.1}{A paragraph}{subsubsection.5.0.1}
\BOOKMARK [1][-]{section.6}{Testing External Jumps}{}
